code/game/area/kutjevo.dm

```diff
@@ -0,0 +1,319 @@
+
+//Areas for the Kutjevo Refinery
+
+/area/kutjevo
+	name = "Kutjevo Refinery"
+	icon = 'icons/turf/area_kutjevo.dmi'
+	//ambience = list('figuresomethingout.ogg')
```

Comment:
?

Reply:
4 years and CM still hasn't figured anything out for Kutjevo's ambience apparently.